orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -67,7 +67,7 @@ public static void inferEvents(Properties props, String pathToConfig, String spe
 		int port = Integer.valueOf(props.getProperty("port"));
 		
 		dbAdaptor = new MySQLAdaptor(host, database, username, password, port);
-		setDbAdaptors(dbAdaptor);
+		setDbAdaptors();
```

Comment:
It may be better to keep the setting of the dbAdaptor in the `setDbAdaptors` method (for both this and other classes) by passing in the created adaptor from the line before.  That way everything to do with setting the adaptor is in one place.

Reply:
Creating multiple adaptors is useful _if_ you plan to have multiple concurrent connections to the database.

Reply:
Kk I've reverted it